BPOR/BPOR.Rms/Controllers/ResearcherController.cs

```diff
@@ -34,6 +34,7 @@ public IActionResult TermsAndConditions(ResearcherTermsAndConditionsViewModel mo
 
     public IActionResult SubmitStudy(ResearcherStudyFormViewModel model)
     {
+        ViewData["Title"] = getPageTitle(model.Step);
```

Comment:
I don't think we should be manipulating `ViewData` here to initialise a multi-step form.
We can do all of this in the view by accessing the model directly, no need to do this twice in the two `SubmitStudy` actions.